pkg/core/native/native_neo.go

```diff
@@ -183,10 +183,22 @@ func (n *NEO) Initialize(ic *interop.Context) error {
 
 // OnPersist implements Contract interface.
 func (n *NEO) OnPersist(ic *interop.Context) error {
+	gas, err := n.GetGASPerBlock(ic, ic.Block.Index)
+	if err != nil {
+		return err
+	}
+	pubs, err := n.GetCommitteeMembers(ic.Chain, ic.DAO)
```

Comment:
We've just optimized out `GetValidatorsInternal` in #1337, maybe we can do the same with this? We can't avoid DB write this time, unfortunately, but at least we can avoid recalculating the committee.

Reply:
Fixed.

Reply:
And this one too.